docs/manage-metadata-using-gravitino.md

```diff
@@ -790,6 +790,22 @@ The following types that Gravitino supports:
 
 The related java doc is [here](pathname:///docs/0.4.0/api/java/com/datastrato/gravitino/rel/types/Type.html).
 
+#### Table column default value
+You can specify a [literal](./expression.md#literal) or an [expression](./expression.md) as the default value when defining a table column. This default value is applied to new rows inserted into the table by the underlying catalog.
+
+When defining a table column, you can specify a [literal](./expression.md#field-reference) or an [expression](./expression.md) as the default value. The default value typically applies to new rows that are inserted into the table by the underlying catalog.
+
+:::note
+When Gravitino loads a table from a catalog that supports default value, if Gravitino is unable to parse the default value, it will use an [Unparsed Expression](./expression.md#unparsed-expression) to preserve the original default value, ensuring that the table can be loaded successfully.
+:::
```

Comment:
I think you should list some of the expressions you supported for column default value. Also point the "unparsed-expression" for others you don't support.

Reply:
Also update the case to add default value support here.

Reply:
fixed